src/StripeTests/Resources/pageable_models/0.json

```diff
@@ -0,0 +1,15 @@
+{
+  "data": [
+    {
+      "id": "pm_123",
+      "object": "pageablemodel"
+    },
+    {
+      "id": "pm_124",
+      "object": "pageablemodel"
+    }
+  ],
+  "has_more": true,
+  "object": "list",
+  "url": "/v1/pageable_models"
```

Comment:
OOC aren't we supposed to have `?starting_after=pm_124` in this URL?

Reply:
No, the API doesn't include `starting_after` parameters in `url` attributes.